Add CopilotKit

suhasdeshpande opened the pull request:
Adding CopilotKit - an open-source framework that enables robust, fast, and customizable AI copilots for your product, with support for multiple languages and frameworks.

Many companies, organizations, and developers use it. The most recent notable milestone is reaching 14k stars on GitHub, demonstrating strong community adoption and trust.

CopilotKit enables developers to build AI copilots fast and customize effortlessly. Whether you're building a code editor, document editor, or any other product that could benefit from AI assistance, CopilotKit provides the building blocks to get you started quickly.

The project is fully open source and available here: [GitHub link](https://github.com/CopilotKit/CopilotKit)

suhasdeshpande commented on ai-enablement-stack.json:
```diff
@@ -579,6 +579,14 @@
           "companies": [
+            {
+              "name": "CopilotKit",
+              "description": "CopilotKit provides a comprehensive AI toolkit for building and deploying AI applications, offering a wide range of tools and services for data processing, model training, and deployment, with support for multiple AI providers and frameworks.",
```
```suggestion
              "description": "AI Copilots for your product. Build fast, customize effortlessly.",
```

nkkko commented:
Hey @suhasdeshpande, it looks nice. It somehow missed my radar. Could you please change the logo to something more visible?

Something like this would be nice, but please keep in mind that the letters should be black, bg transparent and tightly cropped for the best visibility:
![image](https://github.com/user-attachments/assets/f3f85b09-fb71-4519-a185-bcb419863571)

Also, I am looking to serve this as website, see here: https://ai-enablement-stack-production.up.railway.app/

So would be nice to have a slightly stronger description, up to 250 char.

Thanks

suhasdeshpande commented:
> Hey @suhasdeshpande, it looks nice. It somehow missed my radar. Could you please change the logo to something more visible?
> 
> Something like this would be nice, but please keep in mind that the letters should be black, bg transparent and tightly cropped for the best visibility: ![image](https://private-user-images.githubusercontent.com/6250945/397702800-f3f85b09-fb71-4519-a185-bcb419863571.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU4NTY1MzAsIm5iZiI6MTczNTg1NjIzMCwicGF0aCI6Ii82MjUwOTQ1LzM5NzcwMjgwMC1mM2Y4NWIwOS1mYjcxLTQ1MTktYTE4NS1iY2I0MTk4NjM1NzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDEwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMDJUMjIxNzEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGQ0YzgxMzE0NGEwMjU5ODBmMGEwNDlhNzFhZDU2MzUzYjg2NWFkOWU5YzY1ZGRjYjZiNjU5ZjIzMzM4MTkyYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Nwcs7LP763YVj3lAU6GLQ-9JwLCWuvDiPT0SHqEnLoo)
> 
> Also, I am looking to serve this as website, see here: https://ai-enablement-stack-production.up.railway.app/
> 
> So would be nice to have a slightly stronger description, up to 250 char.
> 
> Thanks

Thanks for review. apologies for delay on my end. I believe I addressed your suggestions. Let me know. :)
